matter_server/server/server.py

```diff
@@ -62,13 +62,15 @@ def __init__(
         vendor_id: int,
         fabric_id: int,
         port: int,
-        primary_interface: str | None,
+        listen_addresses: list[str] | None = None,
+        primary_interface: str | None = None,
     ) -> None:
         """Initialize the Matter Server."""
         self.storage_path = storage_path
         self.vendor_id = vendor_id
         self.fabric_id = fabric_id
-        self.port = port
+        self._port = port
```

Comment:
Was this change intended? Technically a breaking change but the attribute is probably not needed for outside use, besides for debugging.

Reply:
My thought was to align it with `listen_address`, which I preferred to be private. I could a read only property instead? :thinking:

Reply:
I'd leave it public